### PR TITLE
Issue/209 unstable test igw

### LINKED ISSUE #209: test_internet_gateway is unstable
From the slack discussion:
The test case test_internet_gateway fails because of a timeout while waiting for the detachment of the vpc.
The teardown of that same test case fails with an `is not attached.`

The timeout for waiting for the correct state should be increased, and the logging also should be improved to see what is happening while waiting.

### PR DESCRIPTION
**Pull request by arnaudsjs**
# Description

Fix unstable test `test_internet_gateway`.

closes #209 

# Self Check:

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Version number is bumped to dev version
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [ ] ~~End user documentation is included or an issue is created for end-user documentation~~

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+v3.0.7
+ - Fix unstable test test_internet_gateway (#209)
```
Comment (edvgui) [inline CHANGELOG.md:2]: Do we specify test changes in CHANGELOG? Isn't this document meant to contain feature upgrades/changes?

Comment (arnaudsjs) [reply CHANGELOG.md:2]: True. But the testing pipeline also runs on the latest release. If we don't release it, the pipeline will keep on failing.

**Review by andraskvr — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (edvgui) [reply CHANGELOG.md:2]: Okay, I understand

**Review by edvgui — APPROVED** [review]

**Comment by arnaudsjs** [comment]
Merged using 982449f04d10936cef50951a0a18ff6f99540457

**Pull request closed without merging**